sql/bis/_init.sql

```diff
@@ -78,9 +88,10 @@ JOBNUMBER_relevant as (
 
     -- Job Type recoding
 	(CASE 
-		WHEN jobtype = 'A1' THEN 'Alteration'
+		WHEN jobtype = 'A1' THEN 'Alteration (A1)'
```

Comment:
Is it possible to leave `'A1' THEN 'Alteration'` and only have `jobtype = 'A2' THEN 'Alteration (A2)'`?  This would allow for consistency between versions.